### PR TITLE
Allow AWS clients to assume role

### PR DESCRIPTION
**Pull request by damianxu88**

### CODE REVIEW

File: orchard-provider-aws/src/main/resources/reference.conf [hunk 1]
```diff
@@ -5,6 +5,9 @@ com.salesforce.mce.orchard {
             accessKeyId = ${?ORCHARD_AWS_ACCESS_KEY_ID}
             secretKey = ${?ORCHARD_AWS_SECRET_KEY}
         }
+        assume.role.credentials {
```
Comment (realstraw) [inline orchard-provider-aws/src/main/resources/reference.conf:8]: this can just be `assume.role.arn`

Comment (damianxu88) [reply orchard-provider-aws/src/main/resources/reference.conf:8]: @realstraw ok, let me change

**Comment by realstraw** [comment]
@damianxu88 can you pull in the latest master?

**Comment by damianxu88** [comment]
@realstraw done and bumped version

**Review by realstraw — APPROVED** [review]

**Pull request merged**